lib-injection/dl_wheels.py

```diff
@@ -39,7 +39,7 @@
     )
 
 
-supported_pythons = ["2.7", "3.6", "3.7", "3.8", "3.9", "3.10", "3.11"]
+supported_pythons = ["3.7", "3.8", "3.9", "3.10", "3.11", "3.12"]
```

Comment:
note that these are supported versions that the script itself supports. It still supports 2.7 and 3.6 we just don't actually use them when calling the script.

Are you sure you want to do away with them here?

Reply:
Leaving this as is won't break anything but it could confuse maintainers. This array implies python<3.7 is supported on the main branch. I also don't see a scenario where we will run this script with changes in 1.x.

Reply:
Okay you win, I will add it back